.github/workflows/pytest.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           cache-suffix: ${{ matrix.python-version }}
           cache-dependency-glob: "**requirements*.txt"
       - name: install-reqs
-        run: uv pip install --upgrade tox virtualenv setuptools -r requirements-dev.txt ibis-framework[duckdb] --system
```

Comment:
So it seems that ibis implemented the dataframe interchange protocol in v6.0.0, however, for such version python 3.9+ is required. Proof: [release notes](https://ibis-project.org/release_notes#6.0.0)